app/src/lang/translations/en-US.yaml

```diff
@@ -1849,7 +1849,7 @@ panels:
     show_data_label: Show Data Label
     group_by: Grouped By
     group_and_function: You must include or omit both a group and a function.
-    y_axix_NaN: The Y axis must be a number.
+    y_axis_NaN: The Y axis must be a number.
```

Comment:
We may need to verify/propagate this change to the cloud exclusive extension (private repo). cc @jaycammarano